packages/dapi-grpc/src/core/client/org.dash.platform.dapi.v0.rs

```diff
@@ -0,0 +1,845 @@
+#[cfg_attr(feature = "serde", derive(::serde::Serialize, ::serde::Deserialize))]
```

Comment:
Did you add these generated files by purpose?

Reply:
nope